socorro/tests/processor/test_processor_app.py

```diff
@@ -25,89 +22,74 @@ def foo(*fargs, **fkwargs):
 
 
 class FakeCrashQueue:
-    def __init__(self, *args, **kwargs):
-        pass
-
     def new_crashes(self):
         return sequencer(
             ((1,), {}), 2, None, ((3,), {})  # ensure both forms acceptable
         )()
 
 
-def get_standard_config():
-    config = DotDict()
-
-    config.source = DotDict()
-    mocked_source_crashstorage = mock.Mock()
-    mocked_source_crashstorage.id = "mocked_source_crashstorage"
-    config.source.crashstorage_class = mock.Mock(
-        return_value=mocked_source_crashstorage
-    )
+@pytest.fixture
+def processor_settings():
+    with settings.override(
+        QUEUE={
+            "class": "socorro.tests.processor.test_processor_app.FakeCrashQueue",
+        },
+        CRASH_SOURCE={
+            "class": "socorro.external.crashstorage_base.InMemoryCrashStorage",
+        },
+        CRASH_DESTINATIONS_ORDER=["dest1"],
+        CRASH_DESTINATIONS={
+            "dest1": {
+                "class": "socorro.external.crashstorage_base.InMemoryCrashStorage",
+            },
+        },
+    ):
+        yield
 
-    config.destination = DotDict()
-    mocked_destination_crashstorage = mock.Mock()
-    mocked_destination_crashstorage.id = "mocked_destination_crashstorage"
-    config.destination.crashstorage_class = mock.Mock(
-        return_value=mocked_destination_crashstorage
-    )
 
-    config.processor = DotDict()
-    mocked_processor = mock.Mock()
-    mocked_processor.id = "mocked_processor"
-    config.processor.processor_class = mock.Mock(return_value=mocked_processor)
-    config.processor.temporary_path = tempfile.gettempdir()
+def get_destination(app, destination_name):
+    index = settings.CRASH_DESTINATIONS_ORDER.index(destination_name)
+    return app.destinations[index]
 
-    config.queue = DotDict()
-    config.queue.crashqueue_class = FakeCrashQueue
 
-    config.companion_process = DotDict()
-    mocked_companion_process = mock.Mock()
-    config.companion_process.companion_class = mock.Mock(
-        return_value=mocked_companion_process
-    )
-
-    return config
+class TestProcessorApp:
```

Comment:
We can probably remove the mocking in these tests since we're using the `InMemoryCrashStorage` and can control what it's got in it.